src/main/java/com/github/juliarn/npc/modifier/TeleportModifier.java

```diff
@@ -11,7 +11,7 @@
  * A teleport modifier for a player
  *
  * @since 2.7-SNAPSHOT
- * @author unldenis <https://github.com/unldenis>
+ * @author <a href="https://github.com/unldenis">unldenis</a>
```

Comment:
If you _really_ want to include the author tag here then just write your name and remove the github profile link...

Reply:
Ok